dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java

```diff
@@ -356,7 +359,7 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
     // Snowflake will refuse (CURRENT_TIMESTAMP - 168 hours) because it is beyond the
     // 7-day window allowed by the server-side function.
     ZonedIntervalIterable intervals =
-        ZonedIntervalIterable.forConnectorArguments(arguments, ChronoUnit.DAYS);
+        ZonedIntervalIterable.forConnectorArguments(arguments, intervalUnit);
```

Comment:
IIUC, it still applies to both query logs and time series. It is okay for now, but it is highly likely that we will have very different time intervals for query logs vs time series in the near future.